bin/distmap

```diff
@@ -270,7 +270,7 @@ GetOptions(
 
 pod2usage(-verbose=>2) if $help;
 
-if ( $hadoop_home ne "" ) {
+if ( defined $hadoop_home and $hadoop_home ne "" ) {
```

Comment:
@robmaz - for whatever reason, while running in my local computer, it fails without the `defined` check...Maybe because it comes from `$ENV{"HADOOP_HOME"}` (can be null?)